src/DbDumperFactory.php

```diff
@@ -20,7 +20,15 @@ public static function createForConnection(string $connectionName): DbDumper
             throw CannotCreateDbDumper::connectionDoesNotExist($connectionName);
         }
 
-        $dbHost = Arr::get($dbConfig, 'read.host', Arr::get($dbConfig, 'host'));
+        $dbHost = Arr::get(
+            $dbConfig,
+            'read.host.0',
+            Arr::get(
```

Comment:
Readability could be improved. Put the fallback in a variable with a good name (`$fallback`?) and use it.

Reply:
I'm not sure I understand the request - do you want me to (using something like the original code) set that value to `$fallback` and then pass that as the second parameter to the `Arr::get`?

Reply:
```php
    $fallback =  Arr::get(
                $dbConfig,
                'read.host',
                Arr::get($dbConfig, 'host'
    );

     $dbHost = Arr::get(
            $dbConfig,
            'read.host.0',
            $fallback,
        );
```

Reply:
Updated, and squashed commits to remove individual commits with StyleCI changes.